blockchain/block.go

```diff
@@ -8,6 +8,15 @@ import (
 	"io"
 )
 
+const (
```

Comment:
hmm thats strange haha, I was wondering why they were added in again. I think you maybe pulled before https://github.com/ubclaunchpad/cumulus/pull/88 was merged because I got rid of all of these.